code/game/sound.dm

```diff
@@ -46,8 +46,11 @@ var/list/trayhit_sound = list('sound/items/trayhit1.ogg', 'sound/items/trayhit2.
 //space sounds have no gas modification, for example. Though >space sounds
 /proc/playsound(var/atom/source, soundin, vol as num, vary = 0, extrarange as num, falloff, var/gas_modified = 1, var/channel = 0,var/wait = FALSE, var/frequency = 0)
 	var/turf/turf_source = get_turf(source)
-
-	ASSERT(!isnull(turf_source))
+	try
+		ASSERT(!isnull(turf_source))
+	catch(var/exception/e)
+		world.log << "[e] on [e.file]:[e.line]"
```

Comment:
Please go read what try/catch is for. This is the same as just removing the assert.

Reply:
You'll have to be explicit with what you were thinking

Reply:
Put the try/catch in the unit test.